qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VerifyLoginSteps.java

```diff
@@ -0,0 +1,104 @@
+        assertTrue(homePage.isConfigureMetadataAppPresent());
+        assertTrue(homePage.isSystemAdministrationAppPresent());
+    }
+
+    @And("user verifies modules available on homepage after login as clerk")
+    public void verifyClerkModulesAvailableOnHomePage() {
+        goToLoginPage().loginAsClerk();
+        assertTrue(homePage.isActiveVisitsAppPresent());
+        assertTrue(homePage.isAppointmentSchedulingAppPresent());
+        assertTrue(homePage.isRegisterPatientCustomizedForRefAppPresent());
+        assertFalse(homePage.isConfigureMetadataAppPresent());
+        assertFalse(homePage.isDataManagementAppPresent());
+        assertFalse(homePage.isSystemAdministrationAppPresent());
+        assertFalse(homePage.isCaptureVitalsAppPresent());
+        assertFalse(homePage.isFindAPatientAppPresent());
+    }
+
+    @And("user verifies modules available on home page after login as doctor")
+    public void verifyDoctorModulesAvailableOnHomePage() {
+        goToLoginPage().loginAsDoctor();
+        assertTrue(homePage.isFindAPatientAppPresent());
+        assertTrue(homePage.isActiveVisitsAppPresent());
+        assertTrue(homePage.isAppointmentSchedulingAppPresent());
+        assertFalse(homePage.isConfigureMetadataAppPresent());
+        assertFalse(homePage.isCaptureVitalsAppPresent());
+        assertFalse(homePage.isRegisterPatientCustomizedForRefAppPresent());
+        assertFalse(homePage.isDataManagementAppPresent());
+        assertFalse(homePage.isSystemAdministrationAppPresent());
+    }
+
+    @And("user verifies modules available on home page after login as nurse")
+    public void verifyNurseModulesAvailableOnHomePage() {
+        goToLoginPage().loginAsNurse();
+        assertTrue(homePage.isFindAPatientAppPresent());
+        assertTrue(homePage.isActiveVisitsAppPresent());
+        assertTrue(homePage.isAppointmentSchedulingAppPresent());
+        assertTrue(homePage.isCaptureVitalsAppPresent());
+        assertFalse(homePage.isRegisterPatientCustomizedForRefAppPresent());
+        assertFalse(homePage.isDataManagementAppPresent());
+        assertFalse(homePage.isSystemAdministrationAppPresent());
+        assertFalse(homePage.isConfigureMetadataAppPresent());
+    }
+
+    @And("user verifies modules available on home page after login as sysadmin")
+    public void verifySysadminModulesAvailableOnHomePage() {
+        goToLoginPage().loginAsSysadmin();
+        assertTrue(homePage.isAppointmentSchedulingAppPresent());
+        assertTrue(homePage.isSystemAdministrationAppPresent());
+        assertFalse(homePage.isFindAPatientAppPresent());
+        assertFalse(homePage.isActiveVisitsAppPresent());
+        assertFalse(homePage.isCaptureVitalsAppPresent());
+        assertFalse(homePage.isRegisterPatientCustomizedForRefAppPresent());
+        assertFalse(homePage.isDataManagementAppPresent());
+        assertFalse(homePage.isConfigureMetadataAppPresent());
```

Comment:
is the system administrator less privileged to `DataManagementApp` and `ConfigureMetadataApp` ?

Reply:
@sherrif10 did you address the above comment from @kdaud ?

Reply:
Yes @kdaud  system administration is less privileged to the two apps.

Reply:
@sherrif10 how can that be?

Reply:
For confirm this. We can use  "sysadmin";  and  "Sysadmin123";